### PR TITLE
Pages and header v2

### PR DESCRIPTION
**Pull request by derekd03**

### CODE REVIEW

**Comment by SamHuo213** [comment]
Please ensure you name branches

features/<descriptionOfChange>

**Comment by SamHuo213** [comment]
I noticed the following issues
1) Multiple Project folder and nav component
2) Top Nav does not highlight based on page

Let's get this in and we can fix it later.

**Review by SamHuo213 — DISMISSED** [review]

File: .vscode/settings.json [hunk 1]
```diff
@@ -14,4 +14,5 @@
   "editor.wordWrap": "on",
   "typescript.updateImportsOnFileMove.enabled": "always",
   "files.eol": "\n",
+  "cSpell.words": ["Parens"]
```
Comment (SamHuo213) [inline .vscode/settings.json:17]: parens?

**Comment by SamHuo213** [comment]
You have a lot of changes that I don't expect. Can you make sure to pull from main.

File: .vscode/settings.json (hunk 1 above)
Comment (derekd03) [reply .vscode/settings.json:17]: Parenthesis, refers to allowParens rule in .prettierrc

Comment (SamHuo213) [reply .vscode/settings.json:17]: ok

**Review by SamHuo213 — APPROVED** [review]

**Comment by umsu2** [comment]
doesn't seem to be up to date to main. @SamHuo213 can you update the branch protection rule to ensure the branch must be up to date to main before allowing branches to be merged.

**Comment by SamHuo213** [comment]
> doesn't seem to be up to date to main. @SamHuo213 can you update the branch protection rule to ensure the branch must be up to date to main before allowing branches to be merged.

I am ok with adding this protection. I am not an admin, but we can quickly talk about it on Tuesday.

There is a dependency on this PR. It's based on a fork and Derek used rebase so it has some issues.

He might need your help to get this PR through. I am incline to get this in ASAP.

**Review by SamHuo213 — APPROVED** [review]

**Pull request merged**